dbms/src/Flash/Executor/PipelineExecutor.cpp

```diff
@@ -139,13 +139,18 @@ RU PipelineExecutor::collectRequestUnit()
 
 Block PipelineExecutor::getSampleBlock() const
 {
+    assert(root_pipeline);
     return root_pipeline->getSampleBlock();
 }
 
 BaseRuntimeStatistics PipelineExecutor::getRuntimeStatistics() const
 {
-    // TODO support getRuntimeStatistics
+    assert(root_pipeline);
+    auto final_plan_exec_id = root_pipeline->getFinalPlanExecId();
```

Comment:
Do we need to check not "" exec_id here?

Reply:
ok, updated.